### DISCUSSION TITLE
ERROR: COULDNT CREATE THE JAVA VIRTUAL MACHINE (Eclipse Adoptium used)

### DISCUSSION THREAD

**Discussion by FurkanVG** [freeplane/freeplane#756] [Help] [answered]
Hey,
just realized my Freeplane is not working.

![image](https://user-images.githubusercontent.com/68749298/196142641-c65b6aef-2cd7-49fa-ae18-637d03f034aa.png)

Versions:
Freeplane: 1.10.4
Java (in this case we use OpenJDK, new name is Eclipse): Eclipse Temurin 8.0.345.1 (x64)

The ENV Variable is set correctly:
FREEPLANE_JAVA_HOME=C:\Program Files\Eclipse Adoptium\jre-8.0.345.1-hotspot

I tried reinstalling Freeplane and Eclipse. Not helping.

Anyne has this issue to?
Thanks.

**Comment by macmarrum** [comment, marked as answer]
Please see
- https://github.com/freeplane/freeplane/issues/661

**Comment by FurkanVG** [comment]
Thanks, idk how i couldnt find that.
